src/server/auditLog.ts

```diff
@@ -0,0 +1,55 @@
+import { getCurrentUser } from "@/server/getCurrentUser";
+import supabase from "@/supabaseClient";
+import { InsertSchema } from "@/databaseUtils";
+import { logErrorReturnLogId } from "@/logger/logger";
+import { DatabaseError } from "@/app/errorClasses";
+
+type AuditLogInsertRecord = InsertSchema["audit_log"];
+export interface AuditLogProps {
+    action?: string;
+    content: Record<string, any>;
```

Comment:
I'm not sure about the any - it could potentially be type ParcelDatabaseInsertRecord for now but it won't cover the content that is not a parcel. What do you think?

Reply:
How about using `Json | null` with `import { Json } from "@/databaseTypesFile";`? since that's what's needed by `AuditLogInsertRecord` anyway?